support-frontend/app/wiring/AppLoader.scala

```diff
@@ -18,25 +19,28 @@ class AppLoader extends ApplicationLoader with StrictLogging {
     EnvironmentVariableCredentialsProvider.create()
   ).build
 
-  private def getParameterStoreConfig(initialConfiguration: Configuration): Configuration = {
-    val identity = AppIdentity.whoAmI(defaultAppName = "support-frontend", CredentialsProvider)
-    val loadedConfig = ConfigurationLoader.load(identity, CredentialsProvider) {
-      case AwsIdentity(app, stack, stage, _) =>
-        val privateConfig = SSMConfigurationLocation(s"/$stack/$app/$stage")
-        val publicConfig = ResourceConfigurationLocation(s"$stage.public.conf")
+  private def getParameterStoreConfig(isDev: Boolean): Try[Configuration] = {
+    for {
+      identity <- if (isDev)
+        Success(DevIdentity("support-frontend"))
```

Comment:
Don't we need to load the local private config as in the case on line 35?

Reply:
Oh I see how it works now, it was a bit tricky to follow in Github

Reply:
yeah it is a bit hard to follow, there may be further imrpovmenets possible!